mobile-app/app/components/TransactionCard.tsx

```diff
@@ -0,0 +1,137 @@
+import { ThemedViewV2, ThemedTextV2, ThemedTouchableOpacityV2 } from '@components/themed'
+import BigNumber from 'bignumber.js'
+import { tailwind } from '@tailwind'
+import { WalletTextInputV2 } from '@components/WalletTextInputV2'
+import { translate } from '@translations'
+import { getNativeIcon } from '@components/icons/assets'
+import { StyleProp, ViewStyle } from 'react-native'
+
+type EditingAmount = 'primary' | 'secondary'
+
+interface TransactionCardProps {
+  symbol: string
+  balance: BigNumber
```

Comment:
```suggestion
  maxValue: BigNumber
```
just to be more generic with the naming